### PR TITLE
chore(ui): replaced buttons with icons on create test form (#470)

### PR DESCRIPTION
**Pull request by rzhekova**
This will resolve issue #470.

Hi guys, I've removed the Cancel button and replaced the other two with icons. Let me know what you think!

### CODE REVIEW

**Comment by enudler** [comment]
hey, @rzhekova 
just one comment
can you move the button to the left and down so it won't be on the scroll bar?
other than that looks great!

**Comment by rzhekova** [comment]
Hi @enudler would you be able to send a screenshot, so I can see what you're seeing? Thanks

**Comment by enudler** [comment]
Sure @rzhekova 

This is how it's now:

Screen Shot 2020-10-08 at 9 02 36

And if possible please move this button to something similar to this:

Screen Shot 2020-10-08 at 9

**Comment by rzhekova** [comment]
Hi @enudler, that's done. Hope it's looking good on your end!

**Comment by enudler** [comment]
@rzhekova looks great!
You are welcome to join to our [Slack workspace](https://join.slack.com/t/predator-dev/shared_invite/enQtNjgwMzE2NjM3MDcyLTg5YTIwMGQyNjZlMjQ4MDNjOTk5YTkwMWYwNzJkOWFmM2QwOGY0ODc3MDU3MWRkYTAwMjRhMjBhOTM1MzFmMjU)

@manorlh please approve

**Comment by manorlh** [comment]
> @rzhekova looks great!
> You are welcome to join to our [Slack workspace](https://join.slack.com/t/predator-dev/shared_invite/enQtNjgwMzE2NjM3MDcyLTg5YTIwMGQyNjZlMjQ4MDNjOTk5YTkwMWYwNzJkOWFmM2QwOGY0ODc3MDU3MWRkYTAwMjRhMjBhOTM1MzFmMjU)
> 
> @manorlh please approve

File: ui/src/features/components/TestForm/index.js [hunk 1]
```diff
@@ -26,10 +25,9 @@ import {
...
   EXPECTATIONS_SPEC_BY_PROP
 } from './constants'
-
-import {
-  faDownload
-} from '@fortawesome/free-solid-svg-icons'
+import IconButton from '../../../components/IconButton';
+import { faDownload } from '@fortawesome/free-solid-svg-icons';
+import { faSave, faPlayCircle } from '@fortawesome/free-regular-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```
Comment (manorlh) [inline ui/src/features/components/TestForm/index.js:32]: @rzhekova  you are awesome!! thank you!!!

**Pull request merged**